test/Table_Tests/src/Common_Table_Operations/Column_Operations_Spec.enso

```diff
@@ -162,12 +162,43 @@ spec setup =
                 (x && y).to_vector . should_equal [True, False, False]
                 (x || y.not).to_vector . should_equal [True, True, True]
 
-        Test.specify "should return null if one of arguments is missing" pending="TODO null handling" <|
-            t = table_builder [["X", [True, False, True]]]
-            x = t.at "X"
-            nulls = [Nothing, Nothing, Nothing, Nothing]
-            (x && Nothing).to_vector . should_equal nulls
-            (x || Nothing).to_vector . should_equal nulls
+        Test.specify "should handle nulls correctly in not" <|
+            t = table_builder [["A", [True, True, True, False, False, False, Nothing, Nothing, Nothing]]]
```

Comment:
any reason these are repeated 3x?

Reply:
as I copied from the && example with the full set.